frontend/src/styles/colors.ts

```diff
@@ -2,8 +2,12 @@
 
 const cornflowerBlue = '#7467EF';
 const white = '#FFFFFF';
+const lightGray = '#d9d9d9';
```

Comment:
Tutaj możesz zachować konwekcję pisania wartości heksadecymalnych z dużych liter(to samo poniżej)